v1.1.1.md

```diff
@@ -0,0 +1,101 @@
+* GUI change: `Network` is renamed to `VM Networks` [[3088](https://github.com/harvester/harvester/issues/3088)]
+
+# Upgrade
+* [Support upgrade from v1.1.0 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+* [Support upgrade from v1.0.3 to v1.1.1](https://docs.harvesterhci.io/v1.1/upgrade/automatic/)
+    * We highly recommend upgrading v1.0.3 clusters to v1.1.1 (skip version v1.1.0).
+    * Due to the implementation changes, all user VMs must shut down during an upgrade. Please stop the VMs before an upgrade.
+
+# Major Bug Fixes
+* Cant's display monitoring dashboard after upgrade, alertmanager, prometheus and grafana monitoring pods Terminating [[#2984](https://github.com/harvester/harvester/issues/2984)]
+* `migrate_harv_mgmt_to_mgmt_br.sh` should remove ClusterNetwork resource [[#3047](https://github.com/harvester/harvester/issues/3047)]
+* Need retry logic around curl in detect_repo [[#3079](https://github.com/harvester/harvester/issues/3079)]
+* Configuring the containerd registry doesn't work anymore during installation [[#3116](https://github.com/harvester/harvester/issues/3116)]
+* Tag rc version, harvester plugin needs to build the corresponding package [[#3145](https://github.com/harvester/harvester/issues/3145)]
+* Upgrade: finished upgrades interfere with newer ones [[#3152](https://github.com/harvester/harvester/issues/3152)]
+* The NICs should keep up after deleting the VLAN config [[#3166](https://github.com/harvester/harvester/issues/3166)]
+* Incomplete dependency chart installation after upgrading 1.0.3 to 1.1.0 and master-head. [[#3168](https://github.com/harvester/harvester/issues/3168)]
+* Upgrade: post-drain job fails when upgrading to the same version [[#3175](https://github.com/harvester/harvester/issues/3175)]
+* Cannot use the secret for all outputs [[#3184](https://github.com/harvester/harvester/issues/3184)]
+
+# Known Issues
+* Windows VMs crashing [[#2522](https://github.com/harvester/harvester/issues/2522)]
+* Unable to provision RKE2 cluster from external Harvester cloud credential [[#2590](https://github.com/harvester/harvester/issues/2590)] (**Deprecated**)
+* Can't completely install Harvester cloud provider on K3s v1.24.4 and v1.23.10 cluster [[#2998](https://github.com/harvester/harvester/issues/2998)]
+* Upgrade
+  * [Multiple nodes pre-drains in an upgrade[3216](https://github.com/harvester/harvester/issues/3216)]
+  * Upgrade stop at upgrading node3 stage, stuck in Pre-drained status [[#3021](https://github.com/harvester/harvester/issues/3021)]
+  * longhorn-system can't be evicted [[#2053](https://github.com/harvester/harvester/issues/2053)]
+  * Upgrade is stuck in "Pre-drained" state (Volume has too many system snapshots) [[#2640](https://github.com/harvester/harvester/issues/2640)]
+  * Upgrade stuck in upgrading first node: Job was active longer than specified deadline [[#2894](https://github.com/harvester/harvester/issues/2894)]
```

Comment:
Do we have workarounds documented for known issues? Need the links here as well.